### DISCUSSION TITLE
Is there any "preferred policy/approach" on new Dashboards?

### DISCUSSION THREAD

**Discussion by hulkster** [jasonacox/Powerwall-Dashboard#381] [General]
I've done some customizations of the Solar-Only Dashboard  - also see #183 - screenshot of mine below. 
I like it because it "fits" on a 4K Display ... but I realize it may not be for everybody.

[In this Reddit thread](https://www.reddit.com/r/TeslaSolar/comments/17hgp8x/solaronly_dashboard/), someone posts a picture of a Dashboard, and in response to someone asking what is is, I say:
 _"That is using [Powerwall DashBoard](https://github.com/jasonacox/Powerwall-Dashboard) ... which ROCKS!"_

Someone then asks _"Can you share your dashboard json?"_ ... which don't mind doing ... but seems like it would be more effective to do here ... BUT ... I'm not sure we want a proliferation of a bazillion dashboards. LOL that User-Interfaces are like opinions ... everyone has one and they are all different ... and think their version is the best!    ;-)

There is also the challenge of what happens when there is a code upgrade to address a bug and/or new feature (such as the Price of Power fix in #58 which I did do)  ... how is that easily applied to the dashboard json in the distribution? And who is "responsible" for doing? 

I'm a newbie here - what do you recommend @jasonacox and others?

![2023_10_14_Solar_DashBoard](https://github.com/jasonacox/Powerwall-Dashboard/assets/145915975/bbc3d629-3145-493c-a07c-43df9d96b86f)

**Comment by jasonacox** [comment]
This is great @hulkster !  

> ... how is that easily applied to the dashboard json in the distribution? And who is "responsible" for doing?

This project is the result of so many people in this community with some who have contributed ideas, suggestions, bug fixes and code snips in issues and discussions like this. 

For your use case, I can think options:  

1. Store it here - Upload the json file here. As the author of this discussion, you can edit your posts and change new versions over time. Downside is that it isn't easy to update by others.
2. Store common use dashboards in the project collection of dashboard.json files at https://github.com/jasonacox/Powerwall-Dashboard/tree/main/dashboards - You could rename your dashboard to `dashboard-solaronly-wide.json` or something descriptive like that and submit a [pull request](https://github.com/jasonacox/Powerwall-Dashboard/pulls) to have it added. 
3. Store your dashboard on [your own repo](https://github.com/hulkster?tab=repositories) and add a link to it here.  As you or others modify it to add features or correct issues, new pull requests could be submitted to your repo to update it for the community.
